checkov/terraform/checks/resource/ncp/AccessControlGroupInboundRule.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.util.type_forcers import force_list
+
+
+class AccessControlGroupInboundRule(BaseResourceCheck):
+    def __init__(self, check_id: str, port: int) -> None:
+        name = f"Ensure no access control groups allow inbound from 0.0.0.0:0 to port {port}"
+        id = check_id
+        supported_resource = [
+            'ncloud_access_control_group_rule',
+        ]
+        categories = (CheckCategories.NETWORKING,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resource)
+        self.port = port
+
+    def scan_resource_conf(self, conf: dict[str, list[Any]]) -> CheckResult:
+
+        if 'inbound' in conf :  # This means it's an ACG resource with inbound block(s)
+            inbound_conf = conf['inbound']
+            for inbound_rule in inbound_conf:
+                for rule in force_list(inbound_rule):
+                    if isinstance(rule, dict) and self.contains_violation(rule):
+                        self.evaluated_keys = [
+                            f'inbound/[{inbound_conf.index(inbound_rule)}]/port_range',
+                            f'inbound/[{inbound_conf.index(inbound_rule)}]/ip_block',
+                        ]
+
+                        return CheckResult.FAILED
+
+        return CheckResult.PASSED
+
+    def contains_violation(self, conf: dict[str, list[Any]]) -> bool:
+        port_range = (conf.get('port_range', [{-1}])[0]).split("-")
```

Comment:
if `port_range` is not in `conf` it will result in a runtime error. I recommend to do something like this
```suggestion
        port_range = conf.get('port_range')
        if isinstance(port_range, list):
            return True/False  # depends on what the defaults are
        port_range = port_range[0].split("-")
```
definitely test it before using my code snippet 😄